cmd/sedge/main.go

```diff
@@ -46,28 +47,29 @@ func main() {
 	// Docker service
 	serviceManager := services.NewServiceManager(dockerClient)
 
+	depsMgr := dependencies.NewDependenciesManager()
```

Comment:
Comment this please

Reply:
Fixed here https://github.com/NethermindEth/sedge/pull/228/commits/0c07e9ce2326e70a6ce0dbc0b5264a9eaacf01f7